schedule/yast/installer_extended/installer_extended.yaml

```diff
@@ -7,90 +7,24 @@ description: >
 schedule:
-  - installation/isosize
-  - installation/data_integrity
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - '{{access_beta_distribution}}'
-  - installation/language_keyboard/switch_keyboard_layout
-  - installation/product_selection/install_SLES
-  - installation/licensing/verify_license_translations
-  - installation/licensing/verify_license_has_to_be_accepted
-  - installation/licensing/accept_license
-  - installation/registration/register_via_scc
-  - installation/module_registration/view_development_versions
-  - installation/module_registration/verify_module_registration
-  - installation/module_registration/skip_module_registration
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_text_mode
-  - installation/partitioning/accept_proposed_layout
-  - installation/releasenotes
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/default_user_simple_pwd
-  - installation/authentication/root_simple_pwd
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-test_data:
-  license:
-    language: English (US)
-    translations:
-      - language: Czech
-        text: Licenční smlouva s koncovým uživatelem
-      - language: English (US)
-        text: End User License Agreement for SUSE Products
-      - language: French
-        text: Contrat de licence utilisateur final
-      - language: German
-        text: Endbenutzer-Lizenzvereinbarung
-      - language: Italian
-        text: Contratto di licenza
-      - language: Japanese
-        text: エンドユーザ使用許諾契約
-      - language: Korean
-        text: SUSE 제품에 관한
-      - language: Portuguese (Brazilian)
-        text: Contrato de Licença para Usuário Final
-      - language: Russian
-        text: Лицензионное соглашение
-      - language: Simplified Chinese
-        text: SUSE 产品
-      - language: Spanish
-        text: Acuerdo de licencia de usuario final
-      - language: Traditional Chinese
-        text: 最終使用者授權合約
-  beta_text: 'You are accessing our Beta Distribution'
-  modules:
-    - sle-ha
-    - sle-module-live-patching
-    - sle-we
-    - sle-module-basesystem
-    - sle-module-certifications
-    - sle-module-containers
-    - sle-module-desktop-applications
-    - sle-module-development-tools
-    - sle-module-legacy
-    - sle-module-public-cloud
-    - sle-module-NVIDIA-compute
-    - sle-module-python3
-    - PackageHub
-    - sle-module-server-applications
-    - sle-module-transactional-server
-    - sle-module-web-scripting
-  registered_modules:
-    - sle-module-basesystem
-    - sle-module-server-applications
+  access_beta:
+    - installation/validate_beta_message
+    - installation/access_beta_distribution
+    - installation/language_keyboard/switch_keyboard_layout
```

Comment:
if now finally is unified and we have only single file for all the architectures, you can move it to test_cases/

Reply:
changed.